src/schema.ts

```diff
@@ -131,44 +164,44 @@ export function build(data: any, opt: QueryConfig = {}): Schema {
   return new Schema(fieldSchemas);
 }
 
-export class Schema {
+export class Schema implements TableSchema {
```

Comment:
Also, when I try

```
interface AI {
  a: string;
}

class A implements AI {
  
}
```
I got an error that `A` doesn't have `a: string`. 

I'm not sure why there is no similar error for schema here.